src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -799,6 +802,9 @@ App::DocumentObjectExecReturn *Helix::execute()
         Standard_Real myRadiusTop = myRadius + myHeight * tan(myAngle/180.0f*M_PI);
 
         this->Shape.setValue(TopoShape().makeSpiralHelix(myRadius, myRadiusTop, myHeight, nbTurns, mySegLen, myLocalCS));
+        GProp_GProps props;
+        BRepGProp::LinearProperties(Shape.getShape().getShape(), props);
+        Length.setValue(props.Mass());

```

Comment:
Are you sure this is correct? Do we get the Mass to get the Length?

Reply:
I think it's ok.
[BRepGProp::LinearProperties Reference](https://dev.opencascade.org/doc/refman/html/class_b_rep_g_prop.html#ab1d4bacc290bfaa8df13dd99ae7b8e70):
> For this computation no linear density is attached to the edges. So, for example, the added mass corresponds to the sum of the lengths of the edges of S.

Reply:
That's consistent with how I researched this, and I couldn't find another way to get the Length.  I agree, it was weird.

Reply:
This could use comment in the code with reference to docs to not confuse other developers.

Reply:
That's a good idea.  Done.